packages/core/helper-plugin/lib/src/components/DynamicTable/tests/Table.test.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable react/prop-types */
+import {
+  BaseCheckbox,
+  Tbody,
+  Td,
+  ThemeProvider,
+  Tr,
+  Typography,
+  lightTheme,
+} from '@strapi/design-system';
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import * as React from 'react';
+import { IntlProvider } from 'react-intl';
+import { MemoryRouter } from 'react-router-dom';
+import TrackingProvider from '../../../providers/TrackingProvider';
+import Table from '../index';
+
+const TableRows = ({
```

Comment:
How come you've chosen to re-write `TableRows`? this feels like we could change our code but this test would still pass?

Reply:
Hi @joshuaellis,
Upon reviewing the layout of the codebase, we have observed that the import flow between the `helper-plugin` package and other packages that contain the `TableRow` component is unidirectional. We want to avoid introducing package circular import unless it's deemed the preferred approach. Noticing that the `TableRows` component was also re-written in the `DynamicTable.stories.mdx` file we opted to adopt the convention.

Furthermore, we have identified that there are currently two TableRow components that could potentially be imported into this test. Would appreciate your input on determining the optimal approach to resolve this situation.

Reply:
Hi @joshuaellis,
Upon reviewing the layout of the codebase, we have observed that the import flow between the `helper-plugin` package and other packages that contain the `TableRow` component is unidirectional. We want to avoid introducing package circular import unless it's deemed the preferred approach. Noticing that the `TableRows` component was also re-written in the `DynamicTable.stories.mdx` file we opted to adopt the convention.

Furthermore, we have identified that there are currently two TableRow components that could potentially be imported into this test. Would appreciate your input on determining the optimal approach to resolve this situation.

Reply:
Oh i see we don't export a TableRow. cool 👍🏼